Connectors/Tekla/Directory.Build.targets

```diff
@@ -1,26 +1,35 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project>
-  <Target AfterTargets="Build"  Name="AfterBuildTekla" Condition="'$(TeklaVersion)' != '' And '$(ContinuousIntegrationBuild)' != 'true'">
+  <Target AfterTargets="Build" Name="AfterBuildTekla" Condition="'$(TeklaVersion)' != '' And '$(ContinuousIntegrationBuild)' != 'true'">
     <ItemGroup>
       <TeklaDlls Include="$(TargetDir)\**\*.*" Exclude="$(TargetDir)*.bmp" />
-      <TeklaBmp Include="$(TargetDir)\Assets\et_element_Speckle.bmp"/>
+      <TeklaBmp Include="$(TargetDir)\Resources\et_element_Speckle.bmp"/>
```

Comment:
would this has any potential side effect on installer (.iss) paths to copy paste correct file/folder?

Reply:
it didn't cause any problems in testing.